src/templates/token.tsx

```diff
@@ -66,28 +84,33 @@ const generateTokenPage = ({ secret }: { secret: string }) => {
           />
           <MjmlAll padding="0" />
         </MjmlAttributes>
-        <MjmlStyle>{css}</MjmlStyle>
+        <MjmlStyle inline>{css}</MjmlStyle>
```

Comment:
Looks like external css is not working for `gmail` at least, that's why we can't use `@media`, btw it has [pretty bad support](https://www.caniemail.com/search/?s=media)
![image](https://user-images.githubusercontent.com/49658988/177816544-6dd2d6a5-765a-468c-a4bd-e42d11c06f91.png)

Also, this is why we can't use `:hover`, if we'll manage to remove `inline`, it would be nice to add some hover effects
![image](https://user-images.githubusercontent.com/49658988/177816676-d1b713a0-a2d4-407a-8479-318e725c1481.png)